warehouse/query-core/src/main/java/datawave/query/jexl/functions/ContentFunctionsDescriptor.java

```diff
@@ -461,15 +464,19 @@ public static JexlNode distributeFunctionIntoIndexQuery(JexlNode function, JexlN
          * @param function
          *            an arbitrary function
          * @param component
-         *            the component
+         *            an ASTAndNode or ASTEqNode
          * @return nothing
          */
         public static JexlNode distributeFunctionIntoComponent(JexlNode function, JexlNode component) {
             List<JexlNode> children = new LinkedList<>();
             String field = findCommonField(component);
             if (field != null) {
                 children.add(updateFunctionWithField(function, field));
-                children.addAll(Arrays.asList(JexlNodes.children(component)));
+                if (component instanceof ASTEQNode) {
+                    children.add(component);
+                } else {
```

Comment:
`else if { ASTAndNode } else { //warn|error}` ?

Reply:
i agree

Reply:
Added in next commit